share/github-backup-utils/ghe-restore-mysql

```diff
@@ -38,11 +38,17 @@ is_binary_backup_feature_on(){
   ghe-ssh "$GHE_HOSTNAME" ghe-config --true "mysql.backup.binary"
 }
 
-if ghe-ssh "$GHE_HOSTNAME" test -f /etc/github/cluster ; then
+ssh_config_file_opt=
+if $CLUSTER ; then
```

Comment:
Is this already set somewhere?

Reply:
Yes, https://github.com/github/backup-utils/blob/473d64b4b1068f42399a80072f40c6330f4fd8ee/bin/ghe-restore#L124